src/FieldWarning/Assets/Units/Waypoint/Pathfinder.cs

```diff
@@ -119,6 +125,11 @@ public bool HasDestination()
         return path.Count > 0;
     }
 
+    public Vector3 GetDestination()
```

Comment:
We generally do this with properties, e.g.:
```
public Vector3 Destination{
  get {
    return path.Count > 0 ? path[0].position : NoPosition;
  }
}
```

The advantage is that the syntax is nicer and more consistent ( all state is retrieved with statements like `dest = Pathfinder.Destination` instead of having some as `dest = Pathfinder.GetDestination()` and others as `bar = Pathfinder.transform.position`).